datadog/resource_datadog_service_level_objective.go

```diff
@@ -122,7 +122,7 @@ func resourceDatadogServiceLevelObjective() *schema.Resource {
 				ValidateDiagFunc: validators.ValidateEnumValue(datadogV1.NewSLOTypeFromValue),
 			},
 			"force_delete": {
-				Description: "A boolean indicating whether this monitor can be deleted even if it’s referenced by other resources (e.g. dashboards).",
+				Description: "A boolean indicating whether this monitor can be deleted even if it's referenced by other resources (e.g. dashboards).",
```

Comment:
```suggestion
				Description: "A boolean indicating whether this monitor can be deleted even if it's referenced by other resources (for example, dashboards).",
```

Reply:
Fixed in [f421d46](https://github.com/DataDog/terraform-provider-datadog/pull/1774/commits/f421d461db0f71dfc3420048e70cf560ee979b8d)